server/test/services/program/ProgramServiceTest.java

```diff
@@ -485,22 +475,6 @@ public void validateProgramDataForCreate_requiresTIListInSelectTiMode() {
             CiviFormError.of("One or more TI Org must be selected for program visibility"));
   }
 
-  @Test
```

Comment:
this test seems to be duplicate with the one above it so I deleted this one. lmk if it was intended tho, I can add it back.

Reply:
The method under test is different, `validateProgramDataForCreate` vs `validateProgramDataForUpdate`

Reply:
sorry my bad, added the `validateProgramDataForUpdate` test back.